autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -322,6 +324,10 @@ def add_model(self, model: AbstractModel, stack_name: str, level: int):
         self.model_fit_times[model.name] = model.fit_time
         self.model_pred_times[model.name] = model.predict_time
         if model.is_valid():
```

Comment:
Not related to this PR, but should the `is_valid()` check happen before updating `model_fit_times` `model_pred_times`?

Reply:
A model must be is_valid to be used in the end, so it is safe to set them before is_valid, since the values will be overwritten when the final version is added that is_valid(). These values are useful if we want to do some model pruning based on fit_time or predict_time of a partially completed bag.